fuzz_testing/test/board/a10_ref/board_env.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<!--
```

Comment:
If the `board/a10_ref`, `hardware/a10_ref_small` and `fake_bsp` are exactly same as the one in the non fuzz test directory. I don't think we should checkin those files again. Can you see if you have anyway to reduce the redundancy here?